app/lib/search_query_builder.rb

```diff
@@ -65,7 +65,23 @@ def metadata_fields
   end
 
   def order_query
-    keywords ? order_by_relevance_query : default_order_query
+    sort_options = finder_content_item.dig('details', 'sort')
+
+    if sort_options.present?
+      sort_option = if params['order']
+                      sort_options.detect { |option| option['name'].parameterize == params['order'] }
+                    else
+                      sort_options.detect { |option| option['default'] } || { 'key' => default_order }
+                    end
```

Comment:
We should probably also catch when someone's used an invalid option for `order` so that it reverts to the default rather than erroring: https://finder-frontend-pr-645.herokuapp.com/news-and-communications?order=crumpets

Reply:
Fixed with 12d03ac584eee7dc47c7c289c0acd1de12a373ba